YatimaStdLib/USize.lean

```diff
@@ -0,0 +1,4 @@
+def USize.next_power_of_two (n : USize) : USize :=
```

Comment:
```suggestion
def USize.nextPowerOfTwo (n : USize) : USize :=
```
Just a naming convention. This will require a change in the test file as well

Reply:
Done